src/errorHandler.ts

```diff
@@ -100,7 +114,7 @@ export function safeExecution(
  */
 export function setupErrorHandler(): Disposable {
     // Setup the statusBarItem
-    statusBarItem = window.createStatusBarItem();
+    statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right, -1);
     statusBarItem.text = 'Prettier';
     statusBarItem.show();
```

Comment:
I think `window.activeTextEditor` and the same thing with score to decide if you show it.